core/tests/fixtures/utils/account/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Page } from '@playwright/test';
+
+import { createAccount } from './create';
+import { deleteAccount } from './delete';
+
+interface CustomerAccount extends Awaited<ReturnType<typeof createAccount>> {
+  cleanup: () => Promise<void>;
+  login: () => Promise<void>;
+  logout: () => Promise<void>;
+}
+
+export class Account {
+  autoCleanup = false;
+  private account: CustomerAccount | null;
+
+  constructor(private readonly page: Page) {
+    this.account = null;
+  }
+
+  async create({ autoCleanup } = { autoCleanup: true }) {
+    const customer = await createAccount();
+
+    this.autoCleanup = autoCleanup;
+    this.account = {
+      ...customer,
+      cleanup: this.cleanup.bind(this),
+      login: this.login.bind(this),
+      logout: this.logout.bind(this),
+    };
+
+    return this.account;
```

Comment:
🍹 we can return `this` here and we can move the account info to the root maybe?

Reply:
sure 👍